packages/server/graphql/public/types/helpers/getActiveTeamCountByOrgIds.ts

```diff
@@ -0,0 +1,76 @@
+import getRethink from '../../../../database/rethinkDriver'
+import {RValue} from '../../../../database/stricterR'
+import TeamMember from '../../../../database/types/TeamMember'
+import {DataLoaderWorker} from '../../../graphql'
+import isValid from '../../../isValid'
+import {InputMaybe} from '../../resolverTypes'
+import {getTeamsByOrgIds} from '../Company'
+
+const THIRTY_DAYS = 1000 * 60 * 60 * 24 * 30
+
+const getActiveTeamCountByOrgIds = async (
```

Comment:
Moved this to its own file so we can use it in Usage Stats and for these org stats